e-school/src/main/java/com/ztp/eschool/entities/User.java

```diff
@@ -13,13 +13,36 @@
 @Setter
 @AllArgsConstructor
 @NoArgsConstructor
+@Builder
 public class User {
+    public static final PasswordEncoder PASSWORD_ENCODER = new BCryptPasswordEncoder();
+
     private @Id
     @GeneratedValue
     Long id;
     private Role role;
     private String username;
+    @JsonIgnore
     private String password;
     private String firstName;
     private String lastName;
+
+    public User(String username, String password, String firstName, String lastName, Role role) {
+        this.role = role;
+        this.username = username;
+        this.setPassword(password);
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
+
+    public void setPassword(String password) {
```

Comment:
Nie jestem pewien czy uzywasz tego jeszcze gdzies? Bo wydaje mi sie, ze nie, ale moglo mi cos umknac? Jezeli jest to tylko tutaj to mozesz usunac i w konstruktorze dodac to kodowanie.